pyramid/config/views.py

```diff
@@ -1545,11 +1588,16 @@ def notfound(request):
         being used, :class:`~pyramid.httpexceptions.HTTPMovedPermanently will
         be used` for the redirect response if a slash-appended route is found.
 
-        .. versionchanged:: 1.6
         .. versionadded:: 1.3
+
+        .. versionchanged:: 1.6
```

Comment:
What exactly changed in 1.6? Wonder if we should add what changed underneath it, otherwise I am not sure that this versionchanged is all that helpful.